websocket-server/noop-persistence-provider.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import type { CRDTDoc } from '@wordpress/sync';
```

Comment:
yjs is already a subdependency, so we should install it explicitly and prefer the "real" type over the alias

```suggestion
import type * as Y from 'yjs';
```

Reply:
Fair point, I've gone ahead and done that.